### PR TITLE
Fix format of pruning doc and analysis doc.

### PR DESCRIPTION
**Pull request by wanghaoshuang**
Fix format of pruning doc and analysis doc.
https://wanghaoshuang.github.io/PaddleSlim/tutorials/sensitivity_demo/

### CODE REVIEW

**Comment by wanghaoshuang** [comment]
image
image
image

File: docs/docs/api/analysis_api.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
-# 模型分析API文档
-
 ## flops
```
Comment (qingqing01) [inline docs/docs/api/analysis_api.md]: flops -> FLOPs

File: docs/docs/api/analysis_api.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
+paddleslim.analysis.flops(program, detail=False) [源代码](https://github.com/PaddlePaddle/PaddleSlim/blob/develop/paddleslim/analysis/flops.py)
 
->paddleslim.analysis.flops(program, detail=False) [源代码]()
-
-获得指定网络的每秒浮点运算次数(FLOPS)。
+: 获得指定网络的每秒浮点运算次数(FLOPS)。
```
Comment (qingqing01) [inline docs/docs/api/analysis_api.md]: 去掉每秒，我们这里并没出真实的计算时间
FLOPs (floating point operations) 
FLOPS (Floating Point Operations Per Second) 

下面注意写法

File: docs/docs/api/analysis_api.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
 
 **参数：**
 
-- **program(paddle.fluid.Program):**  待分析的目标网络。更多关于Program的介绍请参考：[Program概念介绍](https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/fluid_cn/Program_cn.html#program)。
+- **program(paddle.fluid.Program)** - 待分析的目标网络。更多关于Program的介绍请参考：[Program概念介绍](https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/fluid_cn/Program_cn.html#program)。
 
-- **detail(bool):** 是否返回每个卷积层的FLOPS。默认为False。
+- **detail(bool)** - 是否返回每个卷积层的FLOPS。默认为False。
```
Comment (qingqing01) [inline docs/docs/api/analysis_api.md]: https://github.com/PaddlePaddle/PaddleSlim/blob/develop/paddleslim/analysis/flops.py#L21

这里没有包含only_conv的参数

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**